services/src/unit-test/java/uk/gov/moj/sdt/services/TargetApplicationSubmissionServiceTest.java

```diff
@@ -307,225 +394,212 @@ public Object answer() throws Throwable {
-        EasyMock.expect(this.mockCacheable.getValue(IGlobalParameter.class, "TARGET_APP_TIMEOUT")).andReturn(
+        when(this.mockCacheable.getValue(IGlobalParameter.class, TARGET_APP_TIMEOUT)).thenReturn(
                 connectionTimeOutParam);
 
         final IGlobalParameter receiveTimeOutParam = new GlobalParameter();
-        receiveTimeOutParam.setName("TARGET_APP_RESP_TIMEOUT");
-        receiveTimeOutParam.setValue("12000");
-        EasyMock.expect(this.mockCacheable.getValue(IGlobalParameter.class, "TARGET_APP_RESP_TIMEOUT")).andReturn(
+        receiveTimeOutParam.setName(TARGET_APP_RESP_TIMEOUT);
+        receiveTimeOutParam.setValue(TWELVE_THOUSAND);
+        when(this.mockCacheable.getValue(IGlobalParameter.class, TARGET_APP_RESP_TIMEOUT)).thenReturn(
                 receiveTimeOutParam);
 
         final WebServiceException wsException = new WebServiceException("WS Error");
         this.mockConsumerGateway.individualRequest(individualRequest, 1000, 12000);
-        EasyMock.expectLastCall().andThrow(wsException);
-
-        this.mockIndividualRequestDao.persist(individualRequest);
-        EasyMock.expectLastCall();
+        doThrow(wsException).when(mockConsumerGateway).individualRequest(individualRequest, 1000, 12000);
 
-        this.mockMessageWriter.queueMessage(EasyMock.isA(ISdtMessage.class), EasyMock.isA(String.class),
-                EasyMock.eq(true));
-        EasyMock.expectLastCall();
+        this.mockMessageWriter.queueMessage(isA(ISdtMessage.class),isA(String.class), eq(true));
 
-        EasyMock.replay(mockIndividualRequestDao);
-        EasyMock.replay(mockConsumerGateway);
-        EasyMock.replay(mockCacheable);
-        EasyMock.replay(mockMessageWriter);
-        EasyMock.replay(mockErrorMsgCacheable);
+        // Create appender and add to logger
+        ListAppender<ILoggingEvent> listAppender = new ListAppender<>();
+        listAppender.start();
+        logger.addAppender(listAppender);
 
         this.targetAppSubmissionService.processRequestToSubmit(sdtRequestRef);
 
-        EasyMock.verify(mockIndividualRequestDao);
-        EasyMock.verify(mockConsumerGateway);
-        EasyMock.verify(mockCacheable);
-        EasyMock.verify(mockMessageWriter);
-        EasyMock.verify(mockErrorMsgCacheable);
+        List<ILoggingEvent> logList = listAppender.list;
 
-        Assert.assertTrue("Expected to pass", true);
+        assertFalse(logList.isEmpty());
+
+        logger.detachAndStopAllAppenders();
+
+        assertTrue(true,"Expected to pass");
+        verify(mockIndividualRequestDao,times(2)).persist(individualRequest);
     }
 
     /**
      * Test method to test for the soap fault error.
      */
     @Test
     public void processRequestToSubmitSoapFault() {
-        final String sdtRequestRef = "TEST_1";
+        Logger logger = (Logger) LoggerFactory.getLogger(TargetApplicationSubmissionService.class);
+        logger.setLevel(DEBUG);
+        final String sdtRequestRef = TEST_1;
```

Comment:
The sdtRequestRef variable is redundant and can be removed.  The TEST_1 constant can be used in place of it.

Reply:
done